### PR TITLE
Always pass Paths within ""

### LINKED ISSUE #10: On Windows, Red scripts do not Run does not work if Git Bash shell is used as terminal
In Settings,
```
"terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe"
```
Running red scripts does not work because bash shell uses different path separators.
  
Solution -
Pass the paths within `""`

### PR DESCRIPTION
**Pull request by ghost**
Closes #10

### CODE REVIEW

File: src/providers/commandsProvider.ts [hunk 1]
```diff
@@ -94,9 +94,6 @@ function execCommand(currentRedPath: string, command: string, fileUri?: vscode.U
         filePath = fileUri.fsPath;
     }
 
-    if (filePath.indexOf(' ') > 0) {
```
Comment (geekyi) [inline src/providers/commandsProvider.ts:97]: Could have simply removed the `if` condition here. Altho this works about the same

**Review by geekyi — APPROVED** [review]

File: src/providers/commandsProvider.ts (hunk 1 above)
Comment (ghost) [reply src/providers/commandsProvider.ts:97]: @geekyi I had originally done that. But it was giving some issues which i don't remember now 😄.

Comment (geekyi) [reply src/providers/commandsProvider.ts:97]: Yeah, what if it has been already quoted? 🤔

Comment (qtxie) [reply src/providers/commandsProvider.ts:97]: Also, Does this change influence non-Windows platform?

Comment (ghost) [reply src/providers/commandsProvider.ts:97]: @qtxie No idea regarding non-windows platforms since I only have a windows pc.

Comment (munael) [reply src/providers/commandsProvider.ts:97]: It shouldn't, to be honest. In any case, this's the only way to pass paths containing spaces and for that purpose I know it works on Windows, Ubuntu and macOS.

Comment (qtxie) [reply src/providers/commandsProvider.ts:97]: @Enamex Thanks.

**Pull request merged**